src/style.css

```diff
@@ -1,3 +1,66 @@
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+}
+
+nav {
+  display: flex;
+  height: 60px;
+  align-items: center;
+}
+
+ul {
+  padding-left: 0;
+  display: flex;
+  list-style: none;
+  gap: 20px;
```

Comment:
[Requirements](https://github.com/mate-academy/layout_moyo-header?tab=readme-ov-file#requirements): Don't use flex gap property for indents. It's not supported in tests, use margin instead.